packages/flutter_test/lib/src/binding.dart

```diff
@@ -1264,6 +1264,12 @@ enum LiveTestWidgetsFlutterBindingFramePolicy {
   /// on the [SchedulerBinding.hasScheduledFrame] property to determine when the
   /// application has "settled".
   benchmark,
+
+  /// Ignore any request from pump.
+  ///
+  /// This is used for running the test on a device, where scheduling of new
+  /// frames respect what the engine and the device needed.
```

Comment:
It would help to explain how this differs from benchmark and fullyLive, and when I'd want to use this instead of one of those.

Reply:
`benchmark` is actually very different: it's ignoring all animations. But yeah I should update the docs, and restructure the two parts from https://api.flutter.dev/flutter/flutter_test/LiveTestWidgetsFlutterBindingFramePolicy-class.html and
https://api.flutter.dev/flutter/flutter_test/LiveTestWidgetsFlutterBinding/framePolicy.html . But before that I still would like to hear if the general idea is reasonable.